README.md

```diff
@@ -27,11 +27,13 @@ CLI overwrite configuraton loaded from config file so if option defined in confi
 
 ## Basic usage
 
-1. Clone [Jekver repository](https://github.com/kaaproject/jekver) to the directory that contains your project repository with documentation.
+1. [Install Jekyll](https://jekyllrb.com/docs/installation/#ubuntu).
```

Comment:
Please add next steps after "Install Jekyll":
```
gem install bundler
```
Then in  Jekver  root directory (where the "Gemfile" lies) run next to install some needed Jekyll dependencies:
```
bundle install
```